recipes/pyiron_ontology/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pyiron_ontology" %}
+{% set version = "0.0.2" %}
+{% set sha256 = "cdc5630c4e60d4a063211d9a0665aa56a1ff2ffe0c4a8a084d4a76c7b46f71d6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyiron_ontology-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - numpy
+    - openjdk
+    - owlready2
+    - pandas
+    - pint
+    - pyiron_atomistics
+
+test:
+  imports:
+    - pyiron_ontology
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/pyiron/pyiron_ontology
+  summary: pyiron_ontology - module extension to pyiron.
+  license: BSD-3-Clause
+  license_file: LICENSE
+  description: |
+     pyiron_ontology provides owlready2-based ontologies and tools to leverage them for pyiron projects.
+  doc_url: https://pyiron.org
+  dev_url: https://github.com/pyiron
+
+extra:
+  recipe-maintainers:
+    - pyiron-runner
```

Comment:
This looks like a bot account? Does it really need access to merge recipe updates?

Reply:
Yes, it's a bot. Actually, this is a good question. It's on all the other pyiron feedstocks ([e.g.](https://github.com/conda-forge/pyiron_base-feedstock)), and I had assumed it was doing something to handle the really simple version-change bumps. Now that I actually look at it though, this is done by [regro-cf-autotick-bot](https://github.com/regro-cf-autotick-bot) and not pyiron-runner at all -- which makes sense because it's not like there's and scheduled workflows in .github/workflows that use runner, and AFAIK we're not utilizing its access rights with some cron job or anything elsewhere. @jan-janssen, why _do_ we keep adding runner?

Reply:
@liamhuber and @dopplershift : While we originally used the pyiron-runner as bot, it is now more like an administrative account. Basically it should not be used, but if a student leaves the group it gives the group of developers behind pyiron access to the individual sub modules to be able to update them to be compatible to the other pyiron modules if that is necessary.